uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/contact-summary.js

```diff
@@ -14,8 +14,13 @@ module.exports = (function () {
 
       return new Promise(function (resolve) {
         casper.then(function () {
-          casper.click('#manage-roles-and-teams');
+          this.showActions();
+        }.bind(this));
+
+        casper.then(function () {
+          casper.click('[data-contact-access-rights]');
           casper.waitWhileVisible('.spinner');
+
```

Comment:
Not sure why an empty line here

Reply:
same reason why we leave an empty line before the `return`, makes the code more readable

Reply:
OK.